vitrina/datasets/views.py

```diff
@@ -157,7 +161,12 @@ def get(self, request, pk):
         return response
 
 
-class DatasetCreateView(LoginRequiredMixin, PermissionRequiredMixin, CreateView):
+class DatasetCreateView(
+    RevisionMixin,
+    LoginRequiredMixin,
+    PermissionRequiredMixin,
```

Comment:
```suggestion
    LoginRequiredMixin,
    PermissionRequiredMixin,
    RevisionMixin,
```

Permission  checks should always go first.